ConsoleSession.java

```diff
@@ -64,6 +66,10 @@
     private static final String CLEAR = "clear";
     private static final String EXIT = "exit";
     private static final String CLEAN = "clean";
+    private static final String KEYSPACE = "keyspaces";
+
+    // keyspace sub-commands
+    private static String LIST = "list";
```

Comment:
why do we need 2 tokens/commands?

Reply:
This is for extensibility, I imagine we'll want to add `keyspaces create` and `keyspaces delete xxx`? If not, then I'll just collapse these

Reply:
On that note, if we do want to add those commands, I could just add them now. We'll probably want to make it non-plural then.

Reply:
Okay, i see. I don't them being added now. `keyspace delete ...` is definitely useful rather than going into the keyspace to do `clean`.

And yes, make them singular.

Reply:
edit: "I don't `mind` them being added right now".

Reply:
ok that makes more sense!